tree/dataframe/src/RSqliteDS.cxx

```diff
@@ -26,6 +32,328 @@
 #include <cstring>
 #include <stdexcept>
 
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <unistd.h>
+#include <fcntl.h>
+#include <sys/time.h>
+
+#include <linux/limits.h>
```

Comment:
This will be a hassle for portability if Davix receives support on Windows. Please try to keep the code portable as much as possible by using include guards for platform-specific headers. Also, please use alphabetical order for includes if you can.